+vis/patch.m

```diff
@@ -78,7 +78,7 @@
     layer.blending = 'source';
     % Set the layer colour according to the element's colour parameter,
     % adding a fourth, alpha, value
-    layer.maxColour = [newelem.colour 1];
+    layer.maxColour = [newelem.colour(:); 1];
```

Comment:
Did you see my response [here](https://github.com/cortex-lab/Rigbox/issues/143#issuecomment-487624926)?  Also these changes must be consistent across stimulus functions

Reply:
ok, still need to add tests for these stimulus functions and make sure all work when running an exp def. will do this and let you know when done